src/Sector.ts

```diff
@@ -49,6 +49,19 @@ export default class Sector {
     }
   }
 
+  *traverseSectorsBreadthFirst(): IterableIterator<Sector> {
+    const stack: Sector[] = [this];
+    while (stack.length > 0) {
+      const nextSector = stack.shift();
+      nextSector!.children.forEach(child => {
+        if (child !== undefined) {
```

Comment:
Can the child be `undefined`?

Reply:
If so, should we rather do `child != null`?

Reply:
Oops, that's a left-over from debugging.